packages/drupal-remix/README.md

```diff
@@ -73,22 +64,31 @@ export const meta: MetaFunction = ({
   return metaTags({
     tags: data.node.metatag,
     metaTagOverrides: {
-      MetaTagProperty: {
-        "og:url": {
-          kind: "replace",
-          pattern: "drupal-api-url.pantheonsite.io",
-          replacement: "drupal-site.pages.dev",
-        },
+      MetaTagLink: {
+        canonical: "https://your-site-url.com",
       },
     },
   });
 };
 ```
 
-In this example, the `<meta>` tag with the property attribute set to "og:url" will be overridden. The kind property determines the type of override to perform.
+It will produce the following output in the HTML:
 
-MetaTagValue Overrides
-To override a `<meta>` tag with the name attribute, use the MetaTagValue key. For example:
+```html
+<head>
+  <!-- Other meta tags -->
+  <link rel="canonical" href="https://your-site-url.com" />
+  <!-- Other meta tags -->
+</head>
+```
+
+Ignoring the original value of the `canonical` attribute.
+
+#### Search and Replace
+
+To search and replace a specific pattern within the value, you can pass an array of objects that contains the `pattern` and `replacement` keys. It allows you to add multiple search and replace operations to the same value. For example, replace the domain name in the `canonical` attribute of the `<link>` tag.
```

Comment:
Here you're referring to the domain name, but in the example you're including the https schema part too.

Reply:
Nice catch, updated!